### DISCUSSION TITLE
Katas siguientes al modulo 6 aun estan inhabilitadas?...

### DISCUSSION THREAD

**Discussion by Yvillacis** [LaunchX-InnovaccionVirtual/CursoIntroPython#151] [Q&A]
Buenos dias a todos, me gustaria saber si alguno de ustedes ya ve en Teams los modulos posteriores a la Kata 6 "habilitados". . No se si es un problema de mi usuario o general, pero debido al corto tiempo que tengo el fin de semana si me urge fnalizar estos 3 modulos . 

De seguir aun "no disponibles", alguno de ustedes tiene los documentos de estos modulos? El fin de semana anterior si estaban disponibles.....

Se que no quieren asustar a las personas que recien estan ingresando pero es que en mi caso muy particular , si me gustaria avanzar para no estar en problemas el fin de semana. 

De antemano mil gracias por su informacion y ayuda.

**Comment by MIBLAO** [comment]
Buen día, amigx
Yo también tengo el mismo problema, esperemos que en la reunión de hoy nos digan cuál es la situación con las katas siguientes a la 6

**Reply by jcpc91** [reply]
yo también
![imagen](https://user-images.githubusercontent.com/5973152/153682122-9a3aa4a6-de47-4532-bf33-7cdc5fae2254.png)

**Comment by rickygzz** [comment]
En la segunda sesión de Python informaron que aún hay personas ingresando al teams esta semana, por lo que están retrasando la publicación de las katas para que todos vayamos parejo. A esta hora, aún no aparecen katas posteriores a la 6.

Saludos colega

**Comment by Ingrzs** [comment]
Aun no están disponibles , pero trata de ver el live de hoy que es a las 8:00pm (conectate antes porque luego si quieres entrar te dice que esta saturado el servidor, asi me paso a mi) , ahi harán saber nuestras dudas. Saludos

**Comment by oscardanielM** [comment]
Hola

Al parecer no las activaron, por lo que estuve leyendo en las conversaciones de aqui, esperemos hoy en el live menciones algo acerca de tema.

**Comment by JoseAngelSil** [comment]
> Buenos dias a todos, me gustaria saber si alguno de ustedes ya ve en Teams los modulos posteriores a la Kata 6 "habilitados". . No se si es un problema de mi usuario o general, pero debido al corto tiempo que tengo el fin de semana si me urge fnalizar estos 3 modulos .
> 
> De seguir aun "no disponibles", alguno de ustedes tiene los documentos de estos modulos? El fin de semana anterior si estaban disponibles.....
> 
> Se que no quieren asustar a las personas que recien estan ingresando pero es que en mi caso muy particular , si me gustaria avanzar para no estar en problemas el fin de semana.
> 
> De antemano mil gracias por su informacion y ayuda.

Por lo que tengo entendido era ayer, pero si en caso de que no se suban, es probable por los nuevos usuarios o solamente lo entregamos  hasta donde lo tenemos.
